frame/contracts/src/tests.rs

```diff
@@ -458,7 +458,7 @@ fn compile_module<T>(fixture_name: &str) -> wat::Result<(Vec<u8>, <T::Hashing as
 where
 	T: frame_system::Config,
 {
-	let fixture_path = ["fixtures/", fixture_name, ".wat"].concat();
```

Comment:
Wouldn't this break the test when not in debug mode?

Reply:
tested locally I thought it was working, let see if the CI tests go ✅

Reply:
looks like I was wrong 🤔